bindata/network/ovn-kubernetes/001-crd.yaml

```diff
@@ -262,3 +262,195 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: egressrouters.k8s.ovn.org
+spec:
+    group: k8s.ovn.org
```

Comment:
so @danielmellado and I were chatting, and I think the group should probably be `network.operator.openshift.io`

If that's the case, you wouldn't write the CRD directly. Rather, you'd create a struct in `pkg/apis/...`, and `hack/update-codegen.sh` would generate the CRD for you.

Reply:
@squeed : thanks, I've updated the patch accordingly. Just need to clean some pki stuff into another patch :D.